src/admin_extra/service_admin_extra_migration.erl

```diff
@@ -0,0 +1,788 @@
+      2;
+    _ ->
+      0
+  end,
+  case AffDB of
+    0 ->
+      ok;
+    AffDB ->
+      Cols = ["room_id", "luser", "lserver", "aff"],
+      Vals = [ID, LUser, LServer, AffDB],
+      Q = ["INSERT INTO muc_light_occupants ", expand_sql_vals(Cols, Vals), ";"],
+      case mongoose_rdbms:sql_query(?HOST, Q) of
+        {error, Reason} ->
+          io:format("The SQL for 'migrate_muc_light_occupants' has error: ~p for values ~p~n", [Reason, Vals]);
+        _ ->
+          ok
+      end
+  end.
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% convert time for DB
+%% @end
+%% -------------------------------------------------------------------
+-spec to_unixtime(Data :: tuple()) -> Result :: integer().
+
+to_unixtime({Mega, Sec, Micro}) ->
+  Mega * 1000000 * 1000000 + Sec * 1000000 + Micro.
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% Getting all data from table of Mnesia by table name
+%% @end
+%% -------------------------------------------------------------------
+-spec get_all_data(Name :: atom()) -> Result :: {ok, list()} | {error, list()}.
+
+get_all_data(Name) ->
+  try
+    {ok, mnesia:activity(transaction, fun() -> mnesia:select(Name, [{'_', [], ['$_']}]) end)}
+  catch Class:Exception ->
+    {error, io_lib:format("Migration call: [~p:~p/1] catched ~p:~p", [?MODULE, get_all_data, Class, Exception])}
+  end.
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% Generate SQL values: "(field_1, field_1) VALUES ('bear', 'one')"
+%% @end
+%% -------------------------------------------------------------------
+-spec expand_sql_vals(Cols :: list(), Vals :: list()) -> Q :: list().
+
+expand_sql_vals(Cols, Vals) ->
+  EVals = [escape(V) || V <- Vals],
+  ["(", join(Cols, ", "), ") VALUES (", join(EVals, ", "), ")"].
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% Join lists by separator
+%% @end
+%% -------------------------------------------------------------------
+-spec join(list(), list()) -> list().
+
+join([], _Sep) ->
+  [];
+
+join([H|T], Sep) ->
+  [H, [[Sep, X] || X <- T]].
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% escape helper
+%% @end
+%% -------------------------------------------------------------------
+-spec escape(Data :: any()) -> Res :: binary() | list() | any().
+
+escape(Data) when is_integer(Data) ->
+  integer_to_binary(Data);
+
+escape(Data) when is_binary(Data); is_list(Data) ->
+  [$', mongoose_rdbms:escape_characters(Data), $'];
+
+escape(Data) ->
+  Data.
+
+%% -------------------------------------------------------------------
+%% @private
+%% @doc
+%% Convert affiliation atom to integer
+%% @end
+%% -------------------------------------------------------------------
+-spec convert_aff(Aff :: atom()) -> Res :: integer().
+
+convert_aff(Aff) ->
```

Comment:
There is code for this conversation already:

https://github.com/esl/MongooseIM/blob/6b0903f830f7f100b03461c6868d8a7ed7445546/src/pubsub/mod_pubsub_db_rdbms.erl#L711-L717 

It's better to use it, this function is exported.